example/android/build.gradle

```diff
@@ -1,5 +1,5 @@
 buildscript {
-    ext.kotlin_version = '1.5.31'
+    ext.kotlin_version = '1.9.0'
```

Comment:
I'm gonna need some more context around this commit - what exactly has changed so that this now requires kotlin 1.9.0? For context kotlin 1.9.0 was released last week. ideally the version here should be set as low as possible to ensure that the library continues to work with older versions of kotlin

Reply:
This is just an example app. It was failing, because it was explicitly asking to update this version.

https://github.com/ably/ably-flutter/actions/runs/5478663966/jobs/10045729475

Reply:
yeah but what has changed which means we need to upgrade the kotlin version now? presumably it was working before?

Reply:
Because out hunch is, if example app can't accept older versions of kotlin, then our sdk users might face the same issue

Reply:
This is a problem on `main` too - the source is the `fluttertoast` package we use for then example app.

In this PR https://github.com/ponnamkarthik/FlutterToast/pull/429 the minimum Kotlin version was changed. There aren't any tags on GitHub, but if we lock the package to `<=8.2.0` that should fix it :)

Reply:
Okay, so it has nothing to do with the change we made `io.ably:ably-android:1.2.31` right?

Reply:
So, I should change the kotlin version to `8.2.0` or it's fine to keep it as `1.9.0` ?

Reply:
Okay, I have updated the lib version from [pubspec.yaml](https://github.com/ably/ably-flutter/pull/473/files#diff-565b0869896732da3b937c64bc8fd5f0ca37ea1f629d579c29fc70e0f1e3e48e)

Reply:
thanks both 👍